x-pack/legacy/plugins/reporting/server/routes/generate_from_jobparams.ts

```diff
@@ -54,7 +56,8 @@ export function registerGenerateFromJobParams(
     path: `${BASE_GENERATE}/{exportType}`,
     method: 'POST',
     options: getRouteConfig(),
-    handler: async (request: RequestFacade, h: ReportingResponseToolkit) => {
+    handler: async (originalRequest: Legacy.Request, h: ReportingResponseToolkit) => {
+      const request = makeRequestFacade(originalRequest);
```

Comment:
Do we want to keep with the `legacy` naming convention here? `originalRequest` to `legacyRequest`?